src/js/game/LevelMVC/LevelBlock.js

```diff
@@ -209,6 +209,10 @@ module.exports = class LevelBlock {
         this.blockType.startsWith("pressurePlate");
   }
 
+  isRails() {
+    return this.blockType.startsWith("rails") && this.blockType !== "railsRedstoneTorch";
+  }
+
```

Comment:
oh, sorry for my typo; I meant use the existing `isRail` flag

Reply:
d'oh, I totally didn't even see that and assumed we wanted a method for it. 😂 

my bad.